about.html

```diff
@@ -5,8 +5,47 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>About me</title>
+    <!-- Add a favicon -->
+    <link rel= "icon" type= "image/png" href= "/icon.png">
 </head>
 <body>
+      <!-- Add logo/ website title inside <header> -->
+ <header>
+        <h2>website title/logo</h2>
+        <img src= "http://via.placeholder.com/20x20" alt= "logo">
+      <!-- Add navigation bar in <header> [home, project, about me, contact] -->
+    <nav>
+            <a href= "#">Home</a>
+            <a href= "#">Projects</a>
+            <a href= "#">About me</a>
+            <a href= "#">Contact</a>
+        </nav>
+     </header>
+      <!-- create two colums -->
+ 
+      <!-- Add an about me message, social media icons in the first column -->
+     <div class="column" stlye="background-color: ;">
+            <!-- column 1 -->
+            <p>
+            <h2><strong>About me</strong></h2><br>
+            for the love of codes i build things.<br>
+            -----------------------------------------
```

Comment:
Is this supposed to be a comment?

Reply:
They are suppose to be pseucode in form of comments. Explaining my html skeleton